.gitignore

```diff
@@ -114,4 +114,5 @@ admin/static/public/
 admin/webpack-stats.json
 # bower
 admin/static/vendor/
-admin/db.sqlite3
```

Comment:
This is a good idea. This will make you find errors with your migrations much faster.